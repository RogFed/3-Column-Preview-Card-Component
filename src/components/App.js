import React from 'react';

const App = () => (
  <>
    <h1>ReactJS App</h1>
  </>
);

export default App;
