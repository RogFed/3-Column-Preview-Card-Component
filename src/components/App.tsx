import React from 'react'

import PreviewCard from './PreviewCard'

import luxuryIcon from '../images/icon-luxury.svg'
import sedansIcon from '../images/icon-sedans.svg'
import suvsIcon from '../images/icon-suvs.svg'

const cards = [
  {
    image: sedansIcon,
    title: 'sedans',
    text: 'Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city or on your next road trip.',
    bgClass: 'orange',
  },
  {
    image: suvsIcon,
    title: 'suvs',
    text: 'Take an SUV for its spacious interior, power, and versatility. Perfect for your next family vacation and off-road adventures.',
    bgClass: 'cyan',
  },
  {
    image: luxuryIcon,
    title: 'luxury',
    text: 'Cruise in the best car brands without the bloated prices. Enjoy the enhanced comfort of a luxury rental and arrive in style.',
    bgClass: 'darkCyan',
  },
]

const App = () => (
  <>
    <h1>3 Column Preview Card Component</h1>
    <section className='container'>
      {cards.map((card, index) => (
        <PreviewCard {...card} key={index} />
      ))}
    </section>
  </>
)

export default App
