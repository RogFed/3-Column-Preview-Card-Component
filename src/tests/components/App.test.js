import React from 'react';
import { shallow } from 'enzyme';

import App from '../../components/App';

test('Should render App Component', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
