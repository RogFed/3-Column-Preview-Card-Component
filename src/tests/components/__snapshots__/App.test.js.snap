// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render App Component 1`] = `
<Fragment>
  <h1>
    ReactJS App
  </h1>
</Fragment>
`;
